docs/scripting/resources/weaponids.md

```diff
@@ -0,0 +1,71 @@
+| ![](/images/hudIcons/tearGas.png)<br/>Tear Gas                                | ![](/images/deathIcons/death-tearGas.gif)             | WEAPON_TEARGAS            | 17        | 8         | 343           | The coughing effect is disabled in SA-MP                                                                              |
+| ![](/images/hudIcons/molotovCocktail.png)<br/>Molotov Cocktail                | ![](/images/deathIcons/death-molotovCocktail.gif)     | WEAPON_MOLTOV             | 18        | 8         | 344           | Produces fire                                                                                                         |
+| ![](/images/hudIcons/9mm.png)<br/>9mm                                         | ![](/images/deathIcons/death-9mm.gif)                 | WEAPON_COLT45             | 22        | 2         | 346           | Skill can be set with [SetPlayerSkillLevel](../functions/SetplayerSkillLevel)                                         |
+| ![](/images/hudIcons/silenced9mm.png)<br/>Silenced 9mm                        | ![](/images/deathIcons/death-silenced9mm.gif)         | WEAPON_SILENCED           | 23        | 2         | 347           | Skill can be set with [SetPlayerSkillLevel](../functions/SetplayerSkillLevel)                                         |
+| ![](/images/hudIcons/desertEagle.png)<br/>Desert Eagle                        | ![](/images/deathIcons/death-desertEagle.gif)         | WEAPON_DEAGLE             | 24        | 2         | 348           | Skill can be set with [SetPlayerSkillLevel](../functions/SetplayerSkillLevel)                                         |
+| ![](/images/hudIcons/shotgun.png)<br/>Shotgun                                 | ![](/images/deathIcons/death-shotgun.gif)             | WEAPON_SHOTGUN            | 25        | 3         | 349           | Skill can be set with [SetPlayerSkillLevel](../functions/SetplayerSkillLevel)                                         |
+| ![](/images/hudIcons/sawnoffShotgun.png)<br/>Sawnoff Shotgun                  | ![](/images/deathIcons/death-sawnoffShotgun.gif)      | WEAPON_SAWEDOFF           | 26        | 3         | 350           | Skill can be set with [SetPlayerSkillLevel](../functions/SetplayerSkillLevel)                                         |
+| ![](/images/hudIcons/combatShotgun.png)<br/>Combat Shotgun                    | ![](/images/deathIcons/death-combatShotgun.gif)       | WEAPON_SHOTGSPA           | 27        | 3         | 351           | Skill can be set with [SetPlayerSkillLevel](../functions/SetplayerSkillLevel)                                         |
+| ![](/images/hudIcons/microSMG-UZI.png)<br/>Micro SMG/UZI                      | ![](/images/deathIcons/death-microSMG-UZI.gif)        | WEAPON_UZI                | 28        | 4         | 352           | Skill can be set with [SetPlayerSkillLevel](../functions/SetplayerSkillLevel)                                         |
```

Comment:
`/images/deathIcons/death-microSMG-UZI.gif`
`/images/hudIcons/microSMG-UZI.png`
both fail to load